packages/console/src/notebook/ScriptEditor.tsx

```diff
@@ -340,7 +340,7 @@ class ScriptEditor extends Component<ScriptEditorProps, ScriptEditorState> {
     const errorMessage = error ? `Unable to open document. ${error}` : null;
     const editorLanguage = settings ? settings.language ?? null : null;
     const completionProviderEnabled =
-      model && session && editorLanguage === sessionLanguage;
+      model && session != null && editorLanguage === sessionLanguage;
```

Comment:
Shouldn't it be checking `model != null` as well? How is strict boolean not tripping for this?

Reply:
Oh actually this is because strict boolean is ok with `object | null` omitting the check since it's safe. It's not ok w/ boolean, number, string omitting since they are unsafe. Boolean is mostly unsafe if you invert the logic of the variable name and it's mostly to remind you to set a default in that case (at least to me)

`session` is always defined here. Strict boolean expression I guess is smarter now and knows it's always defined, so it's useless in the condition. I added the `!= null` mostly for safety in case our types were wrong and the session is undefined somehow